README.md

```diff
@@ -1,40 +1,110 @@
-QCELL_URL_INITIATE=https://uat-adpelite.qmoney.gm/getMoney
-QCELL_URL_PROCESS=https://uat-adpelite.qmoney.gm/verifyCode
+Before you can begin testing with Qmoney, you need to configure the following variables in your `openimis-be_py/.env` file:
 
+```env
+PSP_QMONEY_GRANTTYPE=<Grant Type>
+PSP_QMONEY_USERNAME=<Authentication Username for Authorization>
+PSP_QMONEY_PASSWORD=<Password for Authentication>
+PSP_QMONEY_AUTH_URL=<URL for Authentication and Access Token Generation>
+PSP_QMONEY_URL_INITIATE=<URL for Initiating Money Transfer>
+PSP_QMONEY_URL_PROCESS=<URL for Verifying Money Transfer
 ```
+Ensure that you replace the placeholders <Grant Type>, <Authentication Username for Authorization>, and other placeholders with the actual values specific to your Qmoney configuration.
 
-* add the following code in your ` openimis-be_py/openIMIS/openIMIS/settings.py `
+These settings are crucial for creating a successful testing environment when working with Qmoney integration.
 
+### Test use case sample
 ```
-QCELL_AUTH_URL = os.environ.get("QCELL_AUTH_URL")
-GRANTTYPE = os.environ.get('GRANTTYPE')
-USERNAME = os.environ.get('USERNAME')
-PASSWORD = os.environ.get('PASSWORD')
-QCELL_URL_PAYMENT = os.environ.get("QCELL_URL_INITIATE")
-QCELL_URL_PROCESS = os.environ.get("QCELL_URL_PROCESS")
+PSP_QMONEY_GRANTTYPE=password
+PSP_QMONEY_USERNAME=14001502
+PSP_QMONEY_PASSWORD=Nhia@123
+PSP_QMONEY_AUTH_URL=https://uat-adpelite.qmoney.gm/login
+PSP_QMONEY_URL_INITIATE=https://uat-adpelite.qmoney.gm/getMoney
+PSP_QMONEY_URL_PROCESS=https://uat-adpelite.qmoney.gm/verifyCode
 ```
 
+
+## Configuration in `openimis-be_py/openIMIS/openIMIS/settings.py`
+
+To integrate Qmoney into your application, add the following code to your `openimis-be_py/openIMIS/openIMIS/settings.py` file:
+
+```python
+PSP_QMONEY_AUTH_URL = os.environ.get("PSP_QMONEY_AUTH_URL")
+PSP_QMONEY_GRANTTYPE = os.environ.get('PSP_QMONEY_GRANTTYPE')
+PSP_QMONEY_USERNAME = os.environ.get('PSP_QMONEY_USERNAME')
+PSP_QMONEY_PASSWORD = os.environ.get('PSP_QMONEY_PASSWORD')
+PSP_QMONEY_URL_PAYMENT = os.environ.get("PSP_QMONEY_URL_INITIATE")
+PSP_QMONEY_URL_PROCESS = os.environ.get("PSP_QMONEY_URL_PROCESS")
+```
+This configuration code is necessary to enable Qmoney integration within your application and should be placed in the specified settings file.
+
+## Creating a Payment Service Provider for Qmoney
+
+To facilitate integration with Qmoney, we've been provided with a test account that allows us to receive money. Below are the test account details:
+
+- `psp_name`: Qmoney
+- `psp_account`: 14001502
+- `psp_pin`: 1234
+
+These details have been included in our migration file. When you run the migration, it will automatically create the Payment Service Provider for Qmoney using the provided test account details. Please note that these details are intended for testing purposes, and if you plan to move to a production environment, you can update them accordingly to reflect the production configuration.
```

Comment:
The reason that was added in the migration file was for test case purpose it can be changed later when going for production. Note those account details are given to us by Qmoney. if i did not answer your question pls explain more in detail

Reply:
The code should not contain any credentials. Please provide way to configure what we need for test or production without anything hard-coded.

Reply:
configuration can be made in The Django Admin by updating the information of payment service provider information

Reply:
those migration file with Qmoney account details was requested to be added by Thibault .

Reply:
I don't remember asking to put credentials in a migration file, that is not a good idea as Christophe explained. 

What I probably said was that if your module needs some data to work, you should have a way of setting up the required data automatically when the module is installed, without having someone go to the Django admin interface and manually adding stuff there. 
For instance, in the 2M Corp grievance module, some ticket category were missing and this could be added in migration files. Not credentials, that's not the same.

Reply:
see the other comment on `mobile_payment/migrations/0002_Qmoney_configuration.py`